src/app/error.js

```diff
@@ -1,7 +1,28 @@
 "use client";
 
+import React from "react";
+import Link from "next/link";
+
 const Error = () => {
-  return <div>Error</div>;
+  return (
+    <>
+      <div className="flex flex-col items-center justify-center min-h-screen">
+        <p className="text-6xl font-bold text-red-600 mb-4">
+          500 -Internal Server Error
+        </p>
+        <p className="text-lg text-gray-700 mb-8">
+          Something went wrong. Please try refreshing the page or go back to the
+          home page:{" "}
+        </p>
```

Comment:
follow the same format as the 404 page, dont need the extra detail

so just a big 500

then "Internal Server Error" as the description